source/win-main/store.ts

```diff
@@ -401,8 +401,14 @@ const config: StoreOptions<ZettlrState> = {
       // that the name did not change (because that would've resulted in a
       // removal and one addition) but rather something else, so we need to make
       // sure to simply re-sort it in case the sorting has changed.
+      // If we have a file, update the parent instead.
       if (ownDescriptor.type === 'directory') {
         Vue.set(ownDescriptor, 'children', sort(ownDescriptor.children, ownDescriptor.sorting))
+      } else if (ownDescriptor.type === 'file') {
+        if (ownDescriptor.parent != null) {
```

Comment:
I would add that to the else if-condition (more style, but it makes the code a tad cleaner)

Reply:
i kept the constant, i think it makes it more readable than reusing ownDescriptor.children.parent everywhere

Reply:
I meant to add the `ownDescriptorparent != null` to the `else if (ownDescriptor.type === 'file')`-condition above.